hieradata/node/auxtel-dc01.ls.lsst.org.yaml

```diff
@@ -0,0 +1,134 @@
+---
+profile::nm::connections:
```

Comment:
Please do not duplicate the interface configuration between the cluster and the nodes. Our pattern has been to put as much network configuration into the cluster layer and a node layer is used only when a host needs configuration which deviates from the majority of the cluster.

Reply:
Independent node configuration applied.

Reply:
There is a node layer file for all of the lsstcam-* nodes that appear to be identical.

Reply:
On CP, all lsstcam nodes belong to cluster lsstcam-ccs; you guys want me to separate them? Override them at node level doesn't work for the unit tests. Let'm know.

Reply:
All dc nodes network configuration were moved into cluster level.